web/dispers/dispers.go

```diff
@@ -16,18 +17,31 @@ CONCEPT INDEXOR'S ROUTES : those functions are used on route ./dispers/conceptin
 
 func createConcept(c echo.Context) error {
 
-	// TODO: Get concept from body
-
-	// TODO: Decrypt concepts
-
-	// TODO: Create array of hashes
-
-	// TODO: Call HashMeThat for each concept
-	hash, err := enclave.HashMeThat(concept)
+	// Get concept from body
+	var in dispers.InputCI
+	if err := json.NewDecoder(c.Request().Body).Decode(&in); err != nil {
+		return c.JSON(http.StatusOK, echo.Map{
+			"ok":    false,
+			"Error": err,
+		})
+	}
 
+	// Create array of hashes
+	hashes := make([]string, len(in.EncryptedConcepts))
+	for index, element := range in.EncryptedConcepts {
+		out := enclave.CreateConcept(element)
+		if out.Error != nil {
```

Comment:
Beware, this is not the standard way to handle errrors in Go. 
It should be `out, err := enclave.CreateConcept(element)`, like [this](https://github.com/cozy/cozy-stack/blob/master/web/data/data.go#L58)

Reply:
Yes, it's true. I only used in CI that kind of way of doing it. I understand totally why it's false. 

I created an out object because of an idea I had in mind but it's the total wrong way of doing it. I know how to do better that now that I got better in Go :)